### PR TITLE
Upgrade subxt to 0.20

### PR DESCRIPTION
**Pull request by kvinwang**
And the metadata files.

### CODE REVIEW

File: standalone/pherry/src/lib.rs [hunk 1]
```diff
@@ -756,7 +756,7 @@ async fn register_worker(
         .tx()
         .phala_registry()
         .register_worker(pruntime_info, attestation)
-        .sign_and_submit_then_watch(signer)
+        .sign_and_submit_then_watch(signer, Default::default())
```
Comment (h4x3rotab) [inline standalone/pherry/src/lib.rs:759]: Not sure if it implies the longevity is "immutable". If so, theoretically we should still set it to a proper value like what we did for `sync_mq_egress`.

**Review by h4x3rotab — APPROVED** [review]
Just left one comment

File: standalone/pherry/src/lib.rs (hunk 1 above)
Comment (kvinwang) [reply standalone/pherry/src/lib.rs]: set

**Review by h4x3rotab — APPROVED** [review]

File: standalone/pherry/src/lib.rs [hunk 2]
```diff
@@ -30,13 +31,16 @@ use phactory_api::blocks::{
...
+use clap::{AppSettings, Parser};
 use msg_sync::{Error as MsgSyncError, Receiver, Sender};
 use notify_client::NotifyClient;
-use clap::{Parser, AppSettings};
-
 
 #[derive(Parser, Debug)]
-#[clap(about = "Sync messages between Phala TEE and the blockchain.", version, author)]
+#[clap(
+    about = "Sync messages between Phala TEE and the blockchain.",
```
Comment (h4x3rotab) [inline standalone/pherry/src/lib.rs:40]:
```suggestion
    about = "Sync messages between pruntime and the blockchain.",
```

**Pull request merged**